source/templates/partials/_nav.twig

```diff
@@ -0,0 +1,38 @@
+<nav class="nav">
+  <a class="avatar" href="#" title="#" aria-label="#"></a>
+
+  <ul class="nav__list">
+    <li class="nav__item">
+      <a class="nav__link" href="#">
+        <img src=""/>
```

Comment:
You prepared the `src` attribute, but there is another important one to add on **every** image in this file :)